src/averageAges.js

```diff
@@ -37,7 +48,20 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const femaleFiltered = people.filter((person) => person.sex === 'f');
```

Comment:
```suggestion
  const femaleFiltered = people.filter((person) => person.sex === 'f');
```
This type of logic is reused in different places here: ```(person) => person.sex === 'm'``` 
Consider creating the function isMan or isWoman to check the person's sex and return true or false